README.md

```diff
@@ -319,11 +319,17 @@ df.writeStream \
 The API Supports a number of options to configure the read
 
 <!--- TODO(#2): Convert to markdown -->
```

Comment:
Please move the changes to README-template.md instead of README.md

Reply:
Done